site/pages/admin/social-media-accounts.page.tsx

```diff
@@ -119,19 +128,29 @@ const SocialMediaAccounts = ({
                 <Link className="govuk-button mt-8 mr-4" data-module="govuk-button" href={twitterAuthUrl}>
                     Connect Twitter
                 </Link>
-                <Link className="govuk-button mt-8" data-module="govuk-button" href={nextdoorAuthUrl}>
-                    Connect Nextdoor
-                </Link>
+                {isOperator ? (
```

Comment:
```suggestion
                {!isOperator ? (
```

Think button was only showing for operator users, should only show for admins :)

Reply:
done